zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/metrics/CamundaExporterMetrics.java

```diff
@@ -146,6 +148,13 @@ public CamundaExporterMetrics(
     Gauge.builder(meterName("since.last.flush.seconds"), this::secondSinceLastFlush)
```

Comment:
❓ out of scope, but I just saw it - there's a `TimeGauge` which makes it easier to distinguish that we're tracking time with specific time units. It's purely API QoL, so I wouldn't push for it, but just FYI.